specification/search/data-plane/Azure.Search/readme.md

```diff
@@ -253,6 +227,14 @@ directive:
           .replace(/(package com.azure.search.models;)/g, "$1\nimport com.fasterxml.jackson.annotation.JsonIgnore;")
           .replace(/(public Document getDocument())/g, "@JsonIgnore\n$1")
 
+    # Make getNextLink and getNextParameterLink package-scope in SearchDocumentsResult
+    - from:
+          - SearchDocumentsResult.java
+      where: $
+      transform: >-
+          return $.replace(/(public SuggestResult setAdditionalProperties)/g, "SuggestResult setAdditionalProperties")
```

Comment:
Why did we make this package scope only? This will require moving this class into the top level namespace as it won't be able to access this property anymore when continuing to the next page.